src/stories/Breadcrumb/Element/Element.stories.ts

```diff
@@ -0,0 +1,27 @@
+import { StoryObj, Meta } from "@storybook/web-components";
+import { html } from "lit";
+
+import { CandyBreadcrumbElementProps } from "../../../components/Breadcrumb/Element";
+import "../../../components/Breadcrumb/Element";
+import { meta } from "./ElementMeta";
+
+type Story = StoryObj<CandyBreadcrumbElementProps>;
+
+const renderElement = (args: CandyBreadcrumbElementProps) => html`<candy-breadcrumb-element
+  label=${args.label}
+  ?isLast=${args.isLast}
+>
+</candy-breadcrumb-element>`;
+
+export default {
+  ...meta,
+  title: "Components/Breadcrumb/Element",
```

Comment:
Rename "Components/Breadcrumb/Element" to "Components/Breadcrumb/Breadcrumb Element"